stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomain.kt

```diff
@@ -147,6 +150,16 @@ public interface ChatDomain {
             return this
         }
 
+        internal fun withIODispatcher(scope: CoroutineDispatcher): Builder {
```

Comment:
I saw in community very popular way to work with different dispatcher. The DispatcherProvider is provided via constructor and you take particular dispatcher from it. We already have it available via name `com.getstream.sdk.chat.infrastructure.DispatchersProvider`. And you can see how it is used in AttachmentsController.